build.fsx

```diff
@@ -233,6 +233,8 @@ Target "RunTests" (fun _ ->
         let rawProjects = match (isWindows) with
                             | true -> !! "./src/**/*.Tests.*sproj"
                                       ++ "./src/**/Akka.Streams.Tests.TCK.csproj"
+                                      -- "src/**/*.Tests.Docker.*sproj"
+                                      -- "src/**/*.*Tests.Docker.*sproj"
```

Comment:
Yep - exactly what I was talking about 👍 
@to11mtm One question - to some people, `Tests.Docker` may not meen that tests will be running under Unix-only systems by default. Should we use this as a convention for Akka.NET that `Tests.Docker` is always something running under Unix only? Or renaming it to `Tests.Unix` or something like that will be more intuitive for other contributors?